framework/doc/content/source/timeintegrators/NewmarkBeta.md

```diff
@@ -0,0 +1,29 @@
+# NewmarkBeta
+
+!syntax description /Executioner/TimeIntegrator/NewmarkBeta
+
+## Description
+
+Newmark time integration [citep:newmark1959amethod] is one of the commonly used time integration methods in structural dynamics problems. In this method, the second ($\ddot{u}$) and first (\dot{u}) time derivatives of a variable $u$ at $t+\Delta t$ are written in terms of the $u$, $\dot{u}$ and $\ddot{u}$ at time $t$, and $u$ at $t+\Delta t$ as shown below:
+
```

Comment:
You are missing open/close `$` signs around the first `\dot{u}` in this line. See also, the rendered docs for this PR [here](http://mooseframework.inl.gov/docs/PRs/12187/site/source/timeintegrators/NewmarkBeta.html#4f68f6b9-de95-4567-8be6-b0035b482a5e).

Reply:
I have corrected this error in the documentation.